src/backend/app/organisations/organisation_routes.py

```diff
@@ -49,6 +49,15 @@ async def get_organisations(
     return await organisation_crud.get_organisations(db, current_user)
 
 
+@router.get("/unapproved/", response_model=list[organisation_schemas.OrganisationOut])
```

Comment:
This already existed before on the "/" endpoint with a flag `approved=True|False`.
I removed that option, as I didn't think we needed to display unapproved orgs via the API.

It's a super admin only thing, so the only possible application is if we have an admin dashboard on the frontend (which doesn't really make sense imo).

Can you think of a use case why we need this code?

Reply:
How do the super admin approve the organisation requests ?
The idea is to display a list of pending organisations, only visible to the super admin. And, then accepts or rejects them.

Reply:
But unless we have an admin dashboard to call the API, this isn't a very user friendly way to approve organisations (having to regularly check the unapproved list).

Ideally the admin receives an email asking for approval, then accepts either by clicking a button (which calls an endpoint), via the approval endpoint, or in the database.

Reply:
If you have been discussing and plan to have an admin dashboard for organisation approval, then this makes sense 👍

Reply:
Yes, the idea is to have a simple dashboard for super admin and the organisation admins, where they can assign new users as the admin, approve/reject the organisations, like those features based on the user roles.

Reply:
Makes sense, thanks for clarifying =)